### ISSUE TITLE
`wildcard_imports` triggers for `use example::bindgen_prelude::*`

### ISSUE THREAD

**Issue by oold** [rust-lang/rust-clippy#10846]
### Summary

Running Clippy on code that has wildcard imports for prelude modules that aren't named exactly as `prelude` triggers the `wildcard_imports` lint.

### Lint Name

wildcard_imports

### Reproducer

I tried this code:

```rust
use napi::{
    bindgen_prelude::*,
};
```

I saw this happen:

```
warning: usage of wildcard import
 --> src\a\mod.rs:5:5
  |
5 |     bindgen_prelude::*,
  |     ^^^^^^^^^^^^^^^^^^ help: try: `bindgen_prelude::{FromNapiValue, ToNapiValue}`
  |
  = help: for further information visit https://rust-lang.github.io/rust-clippy/master/index.html#wildcard_imports
```

I expected to see this happen:

No emitted lint warning.

### Version

```text
rustc 1.69.0 (84c898d65 2023-04-16)
binary: rustc
commit-hash: 84c898d65adf2f39a5a98507f1fe0ce10a2b8dbc
commit-date: 2023-04-16
host: x86_64-pc-windows-msvc
release: 1.69.0
LLVM version: 15.0.7
```

### Additional Labels

@rustbot label +L-pedantic

**Comment by charmitro** [comment]
Is this accepted? If yes, I’d like to give it a try.

**Comment by xFrednet** [comment]
Hey @charmitro, you can work on this one. Usually, we accept most issues, so there is usually no need to ask if one has been accepted. A small tip/note in that regard, we sadly don't have the time to check every notification, so if you want an answer to a question it's good to ping someone from the team. PRs are assigned to someone, there you will always get an answer. :)

**Comment by charmitro** [comment]
> Hey @charmitro, you can work on this one. Usually, we accept most issues, so there is usually no need to ask if one has been accepted. A small tip/note in that regard, we sadly don't have the time to check every notification, so if you want an answer to a question it's good to ping someone from the team. PRs are assigned to someone, there you will always get an answer. :)

ACK. Thanks for the information, assigning it to me then.

@rustbot claim
